python/tvm/tir/schedule/schedule.py

```diff
@@ -710,6 +710,77 @@ def after_rfactor(a: ty.handle, b: ty.handle) -> None:
         """
         return _ffi_api.ScheduleRFactor(self, loop, factor_axis)  # type: ignore # pylint: disable=no-member
 
+    ######## Schedule: Block annotatoin ########
+
+    def storage_align(  # pylint: disable=too-many-arguments
+        self, block: BlockRV, buffer_index: int, axis: int, factor: int, offset: int
+    ) -> None:
+        """Set alignment requirement for specific dimension such that
+           stride[axis] == k * factor + offset for some k.
```

Comment:
Would you like to clarify what `k` is here?

Reply:
There are some TE docs that we could probably add here: https://tvm.apache.org/docs/api/python/te.html#tvm.te.Stage.storage_align